modules/context/api.go

```diff
@@ -367,9 +372,9 @@ func (ctx *APIContext) NotFound(objs ...interface{}) {
 	}
 
 	ctx.JSON(http.StatusNotFound, map[string]interface{}{
-		"message":           message,
-		"documentation_url": setting.API.SwaggerURL,
-		"errors":            errors,
+		"message": message,
+		"url":     setting.API.SwaggerURL,
```

Comment:
Could this be seen as a breaking change? As it's changing a JSON field.

Reply:
Yep, it changes the field name (make the field name same as other API response). IMO no client could use this field to do anything useful, since it is just a URL for a API document, so it should be fine.